### PR TITLE
fix: evac button animation direction

### PR DESCRIPTION
**Pull request by tyler58546**
Partially addresses  #4204

## Summary of Changes

* Fixed EVAC button animation direction (it is now pushed in when on, see #4204)

## Additional context

The EVAC button does not have the correct decals. This will be fixed in a future PR.

## Testing instructions

Make sure the button is pushed in when on.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by tracernz — APPROVED** [review]

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 1]
```diff
@@ -1686,6 +1686,7 @@
                             <LOCK_NODE_ID>LOCK_OVHD_EVAC_COMMAND</LOCK_NODE_ID>
 
                             <TOGGLE_SIMVAR>L:A32NX_EVAC_COMMAND_TOGGLE</TOGGLE_SIMVAR>
+                            <DOWN_CODE>(L:A32NX_EVAC_COMMAND_TOGGLE) !</DOWN_CODE>
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:1689]:
```suggestion
                            <INVERTED/>
```

**Review by davidwalschots — CHANGES_REQUESTED** [review]

**Comment by ChruutvoLuzi** [comment]
Category: QA Tester
Name: ChruutvoLuzi | SD-PB.ch#8902
Date of testing: 11.4.2021
Version of the sim: 1.14.6.0
PR Tested: #4357 
Tier of testing: 1
Changes to observe: Evac button animation Direction


Testing technique: TryOut
Steps you've made to test the Pull Request:
1. Spawned C&D at LSZH
2. Checked EVAC Button

Media:
https://user-images.githubusercontent.com/70195216/114312908-aa03b700-9af4-11eb-8ab2-8ebab25cec8c.mp4

Issues (If Any): None

Overall Rating: Good
Conclusions of the testing: Works as it should, see video.

**Comment by St54Kevin** [comment]
Category: QA Tester
Name: St54Kevin#3583
Date of testing: 12/04/2021
Version of the sim: 1.14.6.0
PR Tested: #4357 
Tier of testing: 1
Changes to observe: EVAC button position

**Steps you've made to test the Pull Request:**
1. Loaded C&D
2. Powered up and checked EVAC button - in when ON

**Media:**

**Issues (If Any):** - 

**Conclusions of the testing:** LGTM

**Comment by davidwalschots** [comment]
Not yet ready to test. Changes requested.

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml (hunk 1 above)
Comment (tyler58546) [reply flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml]: The `INVERTED` parameter only affects the lights on the button. I have created a new parameter called `INVERTED_ANIMATION` for this.

**Review by davidwalschots — APPROVED** [review]

**Comment by marcman86** [comment]
Category: QA Tester
Name: marcman86#4907
Date of testing: 16.04.2021
Version of the sim: 1.15.7.0
PR Tested: #4357
Tier of testing: 1
Changes to observe: EVAC button animation

Testing technique: Tryout

Steps made to test the Pull Request:

1.) Spawned C&D at EDDP
2.) Turn on bat. and ext. power
3.) Test EVAC button

Issues: -

Overall Rating: Good

Conclusions of the testing: Works

Hope this helps :-)

**Comment by FlightSimmer7700** [comment]
Category: QA Tester
Name: FlightSimmer7700#3468
Date of testing: 18/4/21
Version of the sim: 1.15.8.0
PR Tested: #4357 
Tier of testing: 1
Changes to observe:
Evac button

Testing technique: Loaded in and tried
Steps you've made to test the Pull Request:
1. Loaded in 
2. Tried the button

Flight notes (When done a full flight):

Media: Insert screenshots, recordings, or anything relevant to the review here.
https://user-images.githubusercontent.com/70103461/115156858-480bfa00-a07e-11eb-99bf-09766fb94e3a.mp4

Issues (If Any): None
Steps to achieve the issue

Overall Rating: Good
Conclusions of the testing: LGTM

**Pull request merged**